apps/api/src/report/report.service.ts

```diff
@@ -433,46 +514,83 @@ export class ReportService {
   }
 
   /**
-   * Report 10
-   * Get average time spent on each stage and milestone
+   * Report 10 - Average time spent on each stage and their major milestones
    *
-   * NOTE: Counting only hired applicants completed immigration
+   * NNAS
+   *    Applied to NNAS
+   *    Submitted Documents (NNAS Application in Review)
+   *    Received NNAS Report
+   * BCCNM & NCAS
+   *    Applied to BCCNM
+   *    Completed English Language Requirement
+   *    Referred to NCAS
+   *    Completed Computer-Based Assessment (CBA)
+   *    Completed Simulation Lab Assessment (SLA)
+   *    Completed NCAS
+   * Recruitment
+   *    Completed pre-screen (includes both outcomes)
+   *    Completed interview (includes both outcomes)
+   *    Completed reference check (includes both outcomes)
+   *    Hired
+   * Immigration
+   *    Sent First Steps document to candidate
+   *    Sent employer documents to HMBC
+   *    Submitted application to BC PNP
+   *    Received Confirmation of Nomination
+   *    Sent Second Steps document to candidate
+   *    Submitted Work Permit Application
+   *    Immigration Completed
    *
-   * @param t end date of report, YYYY-MM-DD
+   * - Each stage counts applicants who completed the stage.
+   *
+   * @param end date of report, YYYY-MM-DD
+   * @param verbose if true, include the list of sample values
```

Comment:
whats list of sample values?

Reply:
list of values used to calculate mean, mode, and median